resolwe/flow/signals.py

```diff
@@ -13,14 +14,21 @@
 from resolwe.flow.models import Data, Entity
 
 
+def commit_signal():
+    """Nudge manager at the end of every Data save event."""
+    immediate = getattr(settings, 'FLOW_MANAGER_SYNC_DJANGO_SIGNAL', False)
```

Comment:
The name `FLOW_MANAGER_SYNC_DJANGO_SIGNAL` sounds a bit too tied to the implementation. What about `FLOW_MANAGER_RUN_SYNCHRONOUSLY`?

Reply:
Similar as somewhere above, this variable has an extremely specific purpose which should be reflected in the naming. Changing it into `FLOW_MANAGER_RUN_SYNCHRONOUSLY ` would carry false implications of it making every `communicate()` call synchronous and possibly more.